projects/lang-portal/typing-tutor/main.rb

```
@@ -0,0 +1,53 @@
+        self.synth text: text, audio_path: audio_path, client: client, voiceid: voiceid[0]
+      end
+
+      text = t['english']
+      audio_path = File.join(output_path, 'en', "#{text.gsub(/ /,'_')}.mp3")
+      if override && File.exist?(audio_path)
+        puts "English file: #{audio_path} already exists. Skipping synthesis."
+      else
+        self.synth text: text, audio_path: audio_path, client: client, voiceid: voiceid[1]
+      end
+    end # each_with_index
+  end
```

RuboCop (1.69.1)

[convention] Assignment Branch Condition size for run is too high. [<8, 23, 7> 25.34/23]

(Metrics/AbcSize)